specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -10960,6 +11866,10 @@
           "$ref": "#/definitions/StorageProfile",
           "description": "Specifies the storage settings for the virtual machine disks."
         },
+        "capacityReservation": {
+          "$ref": "#/definitions/CapacityReservationProfile",
+          "description": "Specifies information about the capacity reservation that is used to allocate virtual machine. <br><br>Minimum api-version: 2021-04-01."
+        },
```

Comment:
please add this to as the last property of the "VirtualMachineProperties" 

when adding a new subproperty to an existing property, please add it to the bottom

Reply:
updated

Reply:
adding it in the middle changes the signature of the VM creation